app/vmselect/prometheus/prometheus.go

```diff
@@ -501,8 +501,12 @@ var deleteDuration = metrics.NewSummary(`vm_request_duration_seconds{path="/api/
 
 func resetRollupResultCaches() {
 	if len(*selectNodes) == 0 {
-		logger.Panicf("BUG: missing -selectNode flag")
+		logger.Warnf("missing -selectNode flag, cache reset request wont be propagated to the other select nodes." +
+			"you can fix it by adding flag with all vmselect node addresses from your cluster," +
+			" fox example: -selectNode select-addr-1:8481,select-addr-2:8481")
 	}
```

Comment:
should we return from func if no nodes specified?

Reply:
Yes, thought about it, changed.